man/Makefile

```diff
@@ -64,14 +64,26 @@ man: $(man_pages)
 html: $(html_pages)
 pdf: $(pdf_pages)
 
-empty :=
-space := $(empty) $(empty)
-reppat := \<\(${subst ${space},\|,${subst .,(,${addsuffix ),${man_pages}}}}\)
+#
+# Convert the names of man page files to man page citations.
+#
+# ctags.1.rst tags.5.rst ...
```

Comment:
This line should be:
`# ctags.1 tags.5 ...`

Reply:
Thank you. Updated.